### PR TITLE
RM-83168 Release over_react 3.9.0

### PR DESCRIPTION
**Pull request by aaronlademann-wf**
The latest release of over_react includes:

- #620 Consume/update typing for forwardRef2/memo2 functions
- #625 Address analyzer lints / hints

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # OverReact Changelog
 
+## [3.9.0](https://github.com/Workiva/over_react/compare/3.8.1...3.9.0)
+
+- [#620] Consume/update typing for forwardRef2/memo2 functions
```
Comment (greglittlefield-wf) [inline CHANGELOG.md:5]: 1. This should indicate that it's a bugfix it fixes bugs
2. There's another PR in here that needs release notes
```suggestion
- [#620] Fix issue where `uiForwardRef`/`memo` components were being passed JSified props (by consuming [react-dart's `forwardRef2`/`memo2`](https://github.com/cleandart/react-dart/pull/275))
- [#618] Add `makeMapStateToProps`/etc. arguments to `connect` to enable creation of closures specific to component instances
```

**Review by greglittlefield-wf — CHANGES_REQUESTED** [review]
Just a comment on release notes

**Comment by aaronlademann-wf** [comment]
+1

**Review by greglittlefield-wf — APPROVED** [review]

**Comment by greglittlefield-wf** [comment]
QA +1

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**